hack/config-default.sh

```diff
@@ -11,7 +11,10 @@ cdi_namespace=cdi
 image_pull_policy=${IMAGE_PULL_POLICY:-IfNotPresent}
 verbosity=${VERBOSITY:-2}
 package_name=${PACKAGE_NAME:-kubevirt-dev}
-kubevirtci_git_hash="2104301741-c950058"
+kubevirtci_git_hash="2104200713-f58c002"
```

Comment:
something weird about the rebase here ?

Reply:
Done.

Reply:
cant see its done ?
this PR should not touch it (unless git mess around)

Reply:
When I rebase on master, it changes it to this.

Reply:
This is my mistake.
Sorry for the noise. 
I reverted this change.